learning_resources/blockchain.md

```diff
@@ -0,0 +1,32 @@
+# Blockchain Learning Resources
+
+## Bitcoin
+
+As a starting point for learning blockchain Bitcoin has a special place,
+it could be a good reference to understand the idea and basic concepts
+of the blockchain technology
+
+- https://developer.bitcoin.org/devguide/index.html
+
+## Polkadot/Substrate
+
+As an example of Rust developed blockchain and a great successful case of applying
+and idea of sidechains Polkadot and it's ecosystem good to know.
+Also as they have a Rust blockchain framework for building blockchains based on WASM,
+it is also great to explore in that context what we are want to build with hermes.
+
+Polkadot:
+- https://spec.polkadot.network/id-polkadot-protocol
+- https://wiki.polkadot.network/docs/general-index
+
+Substrate (renamed to polkadot-sdk):
+- https://github.com/paritytech/polkadot-sdk
+- https://docs.substrate.io
+
+## Tendermint and Cosmos-SDK
+
+As another reference of blockchain framework with applying sidechain idea but on another language, Go land.
```

Comment:
I'm not sure that Cosmos is a great example of sidechain. Since every chain is independent and communicates with each other (if they want to) using IBC. IBC is just a protocol that allows for different chain in the ecosystem to communicate.

Reply:
I am partially agree with you. Yes IBC protocol it is not exactly about the sidechains it is a general blockchain intercommunication protocol. I think it is good to know some other approaches for solving similar problems